### ISSUE TITLE
clean up the projects page

### ISSUE THREAD

**Issue by mjrb** [usacs/usacs.github.io#61]
its pretty drab

**Comment by mjrb** [comment]
I'm working on this in my fork

**Comment by hemangandhi** [comment]
Yes to everything. The images would be trouble. Maybe icons based on main programming languages used?

**Comment by hemangandhi** [comment]
Also, why aren't the links still red?

**Comment by mjrb** [comment]
i think the h1 tags were over riding the color styling. also i introduced a different css class called project
